.github/workflows/pythonpublish.yml

```diff
@@ -45,6 +45,10 @@ jobs:
       if: "matrix.os != 'ubuntu-latest'"
     - name: Build and publish (Linux)
       uses: RalfG/python-wheels-manylinux-build@v0.3.1
```

Comment:
I've assumed that this builds 3.9 already, but it doesn't. Just up to 3.8 :( Looking into it...